app/schemas/discogs.schemas.ts

```diff
@@ -0,0 +1,14 @@
+import z from "zod";
```

Comment:
I didn't catch this in past PR's I guess, but everything that is not a nextjs specific file (`page.tsx`, `layout.tsx`, etc) shouldn't be in the `app` directory. This `schemas` folder should be in the root of your project, same with your `utils` folder. I'd put the `globals.css` file in a `styles` folder the root of your repo too.